las_geoh5/import_las.py

```diff
@@ -185,8 +188,8 @@ def add_survey(survey: str | Path, drillhole: Drillhole) -> Drillhole:
         file = lasio.read(survey, mnemonic_case="preserve")
         try:
             surveys = np.c_[get_depths(file)["depth"], file["DIP"], file["AZIM"]]
-            if len(drillhole.surveys) == 1:
-                drillhole.surveys = surveys
+            if len(drillhole.surveys) == 1:  # type: ignore
+                drillhole.surveys = surveys  # type: ignore

```

Comment:
@MatthieuCMira has encountered similar issue on his side. We will need to change the inheritance pattern on geoh5py at some point: https://mirageoscience.atlassian.net/browse/GEOPY-1321

Reply:
Actually we have merged GEOPY-1321 already. Can you validate if the type check can be solved now?